### PR TITLE
Add climate constants

### PR DESCRIPTION
**Pull request by raman325**
Similar to #45 since I have also opened https://github.com/home-assistant/core/pull/45177

Note that I did not move https://github.com/home-assistant/core/pull/45177/files#diff-2760c32c24dd7ac020c6a5cf3c42d95953fe65dc6e71a006ce522a638d4f3727R133-R147 because it is so HA specific.

Likely need to rebase after #45 gets merged but assuming both are OK conceptually may as well release a new version with both PRs included

### CODE REVIEW

File: zwave_js_server/const.py [hunk 1]
```diff
@@ -128,3 +129,53 @@ class CommandClass(IntEnum):
...
+    HEATING_ECON = 11
+    COOLING_ECON = 12
+    AWAY = 13
+    FULL_POWER = 15
+    MANUFACTURER_SPECIFIC = 31
+
+
+# In Z-Wave the modes and presets are both in ThermostatMode.
+# This list contains thermostatmodes we should consider a mode only
+MODES_LIST = [
```
Comment (marcelveldt) [inline zwave_js_server/const.py:158]: please rename to THERMOSTAT_MODES_LIST

File: zwave_js_server/const.py [hunk 2]
```diff
@@ -128,3 +129,53 @@ class CommandClass(IntEnum):
...
+    ThermostatMode.OFF,
+    ThermostatMode.HEAT,
+    ThermostatMode.COOL,
+    ThermostatMode.AUTO,
+    ThermostatMode.AUTO_CHANGE_OVER,
+]
+
+# https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/commandclass/ThermostatSetpointCC.ts#L53-L66
+MODE_SETPOINT_MAP: Dict[int, List[str]] = {
```
Comment (marcelveldt) [inline zwave_js_server/const.py:167]: this one too. It is specific for thermostat CC so it should be in the constant's name imo.

File: zwave_js_server/const.py (hunk 1 above)
Comment (marcelveldt) [reply zwave_js_server/const.py:158]: or just THERMOSTAT_MODES (a little shorter)

File: zwave_js_server/const.py (hunk 2 above)
Comment (marcelveldt) [reply zwave_js_server/const.py:167]: BTW: Isn't this also mapping HA modes to ZW modes ? In that case it belongs to the integration.
@MartinHjelmare what do you think ?

Comment (raman325) [reply zwave_js_server/const.py]: IMO it is not - it's mapping enum values (Z-Wave protocol specific) to the values that are needed for ZWave JS (the string versions)

Comment (marcelveldt) [reply zwave_js_server/const.py]: yeah, you're right.

**Review by marcelveldt — APPROVED** [review]
great!

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
Lock PR is merged. Rebase here and we can merge this too.

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
Great!

**Pull request merged**